app.go

```diff
@@ -28,8 +29,14 @@ func init() {
 	caddy.RegisterModule(SouinApp{})
 }
 
+// Provision implements caddy.Provisioner
+func (s SouinApp) Provision(_ caddy.Context) error {
+	return nil
+}
+
```

Comment:
If you're not doing anything in the `Provision` stage, you can remove it. It's not mandatory.